opm/autodiff/StandardWellsDense.hpp

```diff
@@ -116,7 +120,8 @@ enum WellVariablePositions {
                       const std::vector<double>& depth_arg,
                       const std::vector<double>& pv_arg,
                       const RateConverterType* rate_converter,
-                      long int global_nc);
+                      long int global_nc,
+                      const auto& grid);
```

Comment:
It's not even C++17.  It's only supported in the Concepts TS.  Please switch to using a template parameter if you need type deduction.